views/components/LineChart/UPlotLineChart.tsx

```diff
@@ -18,27 +18,45 @@ export default class UPlotLineChartComponent extends React.Component<
   IUPlotLineChartProps,
   IUPlotLineChartState
 > {
+  fmtDate = uPlot.fmtDate("{YYYY}-{MM}-{DD}-{h}:{mm}:{ss}{aa}");
+  localTz = new Intl.DateTimeFormat().resolvedOptions().timeZone;
+  tzDate = ts => uPlot.tzDate(new Date(ts * 1e3), this.localTz);
```

Comment:
Same comment: "Could we declare these varialbes `fmtDate`, `localTz`, and `tzDate` outside the class? Let's have them all as a `const`."